src/widgets/Scrollbar.cpp

```diff
@@ -24,6 +24,16 @@ constexpr qreal SCROLL_DELTA = 5.0;
 
 namespace chatterino {
 
+namespace {
+
+    template <typename A, typename B>
+    bool areClose(const A &a, const B &b)
```

Comment:
I don't think a function like this already exists in our code base... It may be better to move this to some utils file though.

Reply:
We should use [`qFuzzyCompare`](https://doc.qt.io/qt-6/qtnumeric.html#qFuzzyCompare).